composer.json

```diff
@@ -29,6 +29,10 @@
 		"wp-coding-standards/wpcs": "^2.3",
 		"yoast/phpunit-polyfills": "^1.0"
 	},
+	"replace" : {
+		"google/grpc-gcp": "*",
+		"grpc/grpc": "*"
+	},
```

Comment:
❓  Wondering why are we using this syntax instead of just adding the `rm -rf` command into [cleanup-vendor-files.sh](https://github.com/woocommerce/google-listings-and-ads/blob/develop/bin/cleanup-vendor-files.sh)

IMO I think it's better to maintain this cleanups in the sh file to have a single source of truth. Plus the fact that replace `*`doesn't seems very declarative for me. I needed to research in order to understand that '*' will make the package version to match "everything" and hence composer won't install the package.

In any case... if we are using this strategy... then we can get rid of the Line 4 or an output will happen on composer install

```
rm: vendor/google/grpc-gcp/cloudprober/bins/opt/grpc_php_plugin: No such file or directory
```

Reply:
Great catch on the extra line to remove.

I personally think preventing a package from being installed is better than installing and removing. This method of replacing the package is something [we were doing before](https://github.com/woocommerce/google-listings-and-ads/blob/0d68e01fdf8acfa8d3faeccc627b46e82d867f1e/composer.json#L30), so it's not a new concept. We can add a note in the composer.json file if it's helpful to understand what it does.